recipes/genblastg/meta.yaml

```diff
@@ -0,0 +1,23 @@
+about:
+  home: http://genome.sfu.ca/genblast/download.html
+  license: 'GNU'
+  summary: 'genBlast is a program suite, consisting of two programs: genBlastA and genBlastG.
+      genBlastA parses local alignments, or high-scoring segment pairs (HSPs)
+      produced by local sequence alignment programs such as BLAST and WU-BLAST and identify groups of HSPs.'
+
+build:
+  number: 0
+  skip: True # [osx]
+
+package:
+  name: genblastb
+  version: '1.38'
+
+requirements:
+  run:
```

Comment:
They are precompiled binaries.

One, I think genblasta, has a precompiled for osx, but it has a lot of disclaimers. I'll give a try to the source versions and we'll see how that goes. ;-)